dali/dfuplus/dfuplus.cpp

```diff
@@ -1035,28 +1035,12 @@ int CDfuPlusHelper::remove()
         return -1;
     }
 
-    const char* result = resp->getDFUArrayActionResult();
     StringBuffer resultbuf;
-    if(result != NULL && *result != '\0')
```

Comment:
should dfuplus be backwards compatible? Is it possible to use this version of dfuplus pointed at an older version of ESP? Can dfuplus determine the version of the target wsdfu, and act accordingly?